### DISCUSSION TITLE
118 Lecture of Computer Vision

### DISCUSSION THREAD

**Discussion by Agamergen** [mrdbourke/pytorch-deep-learning#951] [General]
What is wrong with this code? I am writing this code along with Mr. Bourke.

![1](https://github.com/mrdbourke/pytorch-deep-learning/assets/170893288/839aeea1-414a-41f0-a4ff-a67bfdd08df6)
![2](https://github.com/mrdbourke/pytorch-deep-learning/assets/170893288/88ea44ed-0d26-45fb-8494-a2ba4c4954db)
![3](https://github.com/mrdbourke/pytorch-deep-learning/assets/170893288/f6347378-f471-4901-9e24-fcdab675f152)
![4](https://github.com/mrdbourke/pytorch-deep-learning/assets/170893288/a4947b7c-42a6-4f8e-abcf-7f5fac6591d3)
![5](https://github.com/mrdbourke/pytorch-deep-learning/assets/170893288/1926891d-b40c-43dd-b37d-a5307979bd54)
![6](https://github.com/mrdbourke/pytorch-deep-learning/assets/170893288/c3ae627b-a4b0-4813-95ed-b587d9fe5714)

**Comment by aldragan0** [comment]
From the looks of it (in the first image), it seems to be an indentation issue. Your `forward` function is outside the class.
